doc/whats_new.rst

```diff
@@ -146,6 +146,7 @@ API
 
     - Deprecated :func:`mne.time_frequency.cwt_morlet` and :func:`mne.time_frequency.single_trial_power` in favour of :func:`mne.time_frequency.tfr_morlet` with parameter average=False, by `Jean-Remi King`_ and `Alex Gramfort`_
 
+    - Added :class:`mne.decoding.TemporalFilter` with same functionality as :class:`mne.decoding.FilterEstimator` however with explicit passing of `sfreq` parameter as well as compatibility with scikit-learn Pipeline by `Asish Panda`_
```

Comment:
rephrase to:
- Added :class:`mne.decoding.TemporalFilter` to filter temporal data in scikit-learn pipelines, by `Asish Panda`_